base_layer/core/src/chain_storage/metadata.rs

```diff
@@ -133,6 +133,18 @@ impl Display for ChainMetadata {
     }
 }
 
+impl Default for ChainMetadata {
```

Comment:
I actually removed this default in a previous PR. Having the default led a number of checks to see if the values made sense. In cases where the data is not available, I used an Option instead. In test code, I'd suggest specify mock values.